src/libraries/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -29,10 +29,10 @@ public partial class EnumerableQuery<T> : System.Linq.EnumerableQuery, System.Co
         System.Collections.Generic.IEnumerator<T> System.Collections.Generic.IEnumerable<T>.GetEnumerator() { throw null; }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         System.Linq.IQueryable System.Linq.IQueryProvider.CreateQuery(System.Linq.Expressions.Expression expression) { throw null; }
-        System.Linq.IQueryable<S> System.Linq.IQueryProvider.CreateQuery<S>(System.Linq.Expressions.Expression expression) { throw null; }
```

Comment:
Why do we need this update?

Reply:
This as all generated code. I'm assuming what happened here is somebody changed the type argument name but didn't regenerate the ref sources. I've seen instances of this elsewhere too.